src/lib.rs

```diff
@@ -15,7 +15,7 @@ use std::{
   sync::{Arc, Mutex},
 };
 
-const STATE_FILENAME: &str = ".window-state";
+pub const STATE_FILENAME: &str = ".window-state";
```

Comment:
why does this need to be public?

Reply:
This is a temporary solution that allows users to delete state.

Reply:
I guess it is fine to expose it.